frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -546,7 +549,19 @@ export default function ToolbarView() {
   const disabledInfoForCurrentTool = disabledInfosForTools[activeTool];
   useEffect(() => {
     if (disabledInfoForCurrentTool.isDisabled) {
+      setLastForcefulDisabledTool(activeTool);
       Store.dispatch(setToolAction(AnnotationToolEnum.MOVE));
+    } else if (
+      lastForcefulDisabledTool != null &&
+      !disabledInfosForTools[lastForcefulDisabledTool].isDisabled &&
+      activeTool === AnnotationToolEnum.MOVE
+    ) {
+      // Reenable the tool that was disabled before.
+      setLastForcefulDisabledTool(null);
+      Store.dispatch(setToolAction(lastForcefulDisabledTool));
+    } else if (activeTool !== AnnotationToolEnum.MOVE) {
+      // Forget the last disabled tool as another tool besides the move tool was selected.
+      setLastForcefulDisabledTool(null);
     }
   }, [activeTool, disabledInfoForCurrentTool]);
```

Comment:
For completeness sake, `lastForcefulDisabledTool` should be included in the dependencies here as well, right?